spec/xcodebuild_analyze_action_project_spec.rb

```diff
@@ -0,0 +1,48 @@
+describe Fastlane::Actions::XcodebuildAnalyzeAction do
+  describe 'xcodebuild analyze should run xcodebuild with -project' do
+    before(:each) do
+      expect(Fastlane::Actions)
+        .to receive(:sh)
+        .with("xcodebuild -project Meniga.xcodeproj -scheme Meniga analyze")
+    end
+
+    it 'when project provided and workspace not provided' do
+      Fastlane::FastFile.new.parse("lane :test do
```

Comment:
What do you think about formatting it like this? 

```
Fastlane::FastFile.new.parse("
    lane :test do
      xcodebuild_analyze(project: 'Meniga.xcodeproj', scheme: 'Meniga')
    end
").runner.execute(:test)
```

Reply:
I changed format and extract this to new method.